cmd/crowdsec-cli/dashboard.go

```diff
@@ -72,7 +76,48 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
 			if metabasePassword == "" {
 				metabasePassword = generatePassword(16)
 			}
-			mb, err := metabase.SetupMetabase(csConfig.API.Server.DbConfig, metabaseListenAddress, metabaseListenPort, metabaseUser, metabasePassword, metabaseDbPath)
+			var answer bool
+			groupExist := false
+			dockerGroup, err := user.LookupGroup(crowdsecGroup)
+			if err == nil {
+				groupExist = true
+			}
+			if !forceYes && !groupExist {
+				prompt := &survey.Confirm{
+					Message: fmt.Sprintf("We need to add a new group called '%s' to the system, is it ok for you ?", crowdsecGroup),
```

Comment:
we need a more explicit message, ie. "for metabase docker to be able to access SQLite file, we need to add a group called ..."